thirdparty/README.md

```diff
@@ -35,6 +35,19 @@ Files extracted from upstream source:
 - Version: git (8b263a18fca98ea371e54227837321c5cdaa1ba7, 2021)
 - License: MPL 2.0
 
+File extracted from a recent Fedora install:
+/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
+(It can't be extracted directly from the package,
+as it's generated on the user's system.)
```

Comment:
I think this is from a rebase issue?

Reply:
I resolved this issue because I think the original bundle is required for the fallback.